api/CSSTransition.json

```diff
@@ -0,0 +1,146 @@
+{
+  "api": {
+    "CSSTransition": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "75"
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": "13.1"
+          },
+          "safari_ios": {
+            "version_added": "13.4"
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "CSSTransition": {
+        "__compat": {
+          "description": "<code>CSSTransition()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
```suggestion
              "version_added": "78"
```

Reply:
@vinyldarkscratch There are three resolved, but unapplied suggestions here. Was this a mistake (maybe relating to the revert) or was there more information that conflicted with version 78 for these entries?

Reply:
The lack of merge was a mistake -- I forgot to commit the suggestions when adding the constructor code back.  Fixed now!